File: ash2/ash/Service/ITb_userService.cs
using System;
using System.Collections;
using model;

namespace ash.Service
{
    public interface ITb_userService
    {
        void test();

        Tb_user get(int id);

        void save(Tb_user model);

        void update(Tb_user model);

        void delete(Tb_user model);

        IList getAll();
    }
}

File: ssh/src/service/interf/ITb_userService.java
package service.interf;

import java.util.List;

import model.test.Tb_user;

public interface ITb_userService {
	public Tb_user getById(int id);

	public void save(Tb_user model);

	public void update(Tb_user model);

	public void delete(Tb_user model);

	public void deleteById(int id);

	@SuppressWarnings("unchecked")
	public List getAll();
}
